<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1133</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Abonne</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1091</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabBarAutoHide">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="Abonne">
     <attribute name="title">
      <string>Abonne</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1081</width>
        <height>521</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1081</width>
          <height>581</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(85, 170, 255);
background-color: rgb(170, 255, 255);</string>
        </property>
        <property name="title">
         <string>Ajout</string>
        </property>
        <widget class="QLineEdit" name="le_id">
         <property name="geometry">
          <rect>
           <x>32</x>
           <y>40</y>
           <width>171</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 170, 127);
background-color: rgb(216, 193, 255);</string>
         </property>
         <property name="placeholderText">
          <string>Id</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_nom">
         <property name="geometry">
          <rect>
           <x>32</x>
           <y>100</y>
           <width>171</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(216, 193, 255);</string>
         </property>
         <property name="placeholderText">
          <string>Nom</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_prenom">
         <property name="geometry">
          <rect>
           <x>32</x>
           <y>160</y>
           <width>171</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(216, 193, 255);
</string>
         </property>
         <property name="placeholderText">
          <string>Prenom</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_ajouter">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>70</y>
           <width>131</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_payement">
         <property name="geometry">
          <rect>
           <x>32</x>
           <y>220</y>
           <width>171</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(216, 193, 255);</string>
         </property>
         <property name="placeholderText">
          <string>Payement</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_nbr_chats">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>290</y>
           <width>171</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(216, 193, 255);</string>
         </property>
         <property name="placeholderText">
          <string>Nbr_Chats</string>
         </property>
        </widget>
        <widget class="QPushButton" name="le_Modifier">
         <property name="geometry">
          <rect>
           <x>540</x>
           <y>70</y>
           <width>131</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Modifier</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>100</y>
           <width>131</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>      Trier par:</string>
         </property>
        </widget>
        <widget class="QTableView" name="tableView">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>140</y>
           <width>541</width>
           <height>151</height>
          </rect>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>310</y>
           <width>97</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Id</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>310</y>
           <width>97</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Nom</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_3">
         <property name="geometry">
          <rect>
           <x>480</x>
           <y>310</y>
           <width>97</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Prenom</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_recherche_abonne">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>20</y>
           <width>241</width>
           <height>22</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>recherche</string>
         </property>
        </widget>
        <widget class="QPushButton" name="le_stat">
         <property name="geometry">
          <rect>
           <x>730</x>
           <y>310</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>stat</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_pdf">
         <property name="geometry">
          <rect>
           <x>630</x>
           <y>310</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Pdf</string>
         </property>
        </widget>
        <widget class="QComboBox" name="le_sexe">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>350</y>
           <width>121</width>
           <height>22</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>male</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>femelle</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>350</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 0, 127);
background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Sexe   :</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Affichier</string>
       </attribute>
       <widget class="QTableView" name="tab_etudiant">
        <property name="geometry">
         <rect>
          <x>25</x>
          <y>10</y>
          <width>1011</width>
          <height>441</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Supprimer </string>
       </attribute>
       <widget class="QLineEdit" name="le_id_sup">
        <property name="geometry">
         <rect>
          <x>52</x>
          <y>60</y>
          <width>171</width>
          <height>20</height>
         </rect>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Identifiant</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_supprimer">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>90</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Abonnement</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1061</width>
        <height>511</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QPushButton" name="pb_ajouter_2">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>110</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>140</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Date Debut</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>210</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Date Fin</string>
        </property>
       </widget>
       <widget class="QPushButton" name="le_modifier_2">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>180</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_id_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>80</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
       <widget class="QDateEdit" name="le_date_deb">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>140</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateEdit" name="le_date_fin">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>210</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>80</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Id</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Afficher</string>
       </attribute>
       <widget class="QTableView" name="tab_etudiant_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1061</width>
          <height>621</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Supprimer</string>
       </attribute>
       <widget class="QLineEdit" name="le_id_sup_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>70</y>
          <width>151</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Identifiant</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_supprimer_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>100</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1133</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
